### PR TITLE
#187 - Added unassigned() method in Tasks.

### PR DESCRIPTION
**Pull request by criske**
PR for #187 

Implemented and tested UnassignedTasks.

Left puzzle to implement and test unassigned() for ProjectTasks and InMemoryTasks.

### CODE REVIEW

**Comment by criske** [comment]
I think if fails because of Github's request rate limit.

File: self-api/src/main/java/com/selfxdsd/api/Tasks.java [hunk 1]
```diff
@@ -71,4 +71,10 @@ Tasks ofProject(
     Tasks ofContributor(final String username,
                         final String provider);
 
+    /**
+     * Get the unassigned tasks.
+     * @return Tasks
```
Comment (amihaiemil) [inline self-api/src/main/java/com/selfxdsd/api/Tasks.java:76]: @criske Can you put a dot after `Tasks`, please

File: self-core-impl/src/main/java/com/selfxdsd/core/tasks/ContributorTasks.java [hunk 2]
```diff
@@ -100,6 +100,12 @@ public Tasks ofContributor(final String username,
         return tasks;
     }
 
+    @Override
+    public Tasks unassigned() {
+        throw new UnsupportedOperationException("Contributors can't have "
+            + " unassigned tasks");
```
Comment (amihaiemil) [inline self-core-impl/src/main/java/com/selfxdsd/core/tasks/ContributorTasks.java:106]: @criske Can you give a more detailed message: "These are the Tasks or Contributor X from provider Y, no unassigned Tasks here.".

File: self-core-impl/src/main/java/com/selfxdsd/core/tasks/UnassignedTasks.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+import java.util.stream.Collectors;
+
+/**
+ * Unassigned Tasks. This class <b>just represents</b>
+ * the tasks. The actual filtering has to be done in an upper layer.
+ *
+ * @author criske
+ * @version $Id$
+ * @since 0.0.5
+ * @todo #190: Implement and test unassigned() for ProjectTasks
```
Comment (amihaiemil) [inline self-core-impl/src/main/java/com/selfxdsd/core/tasks/UnassignedTasks.java:20]: @criske You forgot the ``:30min`` here :D

**Comment by amihaiemil** [comment]
@rultor merge it

**Pull request merged**